package.json

```diff
@@ -18,6 +18,7 @@
         "version": "npm run build",
         "postinstall": "npm run bootstrap",
         "build": "npm run bootstrap && lerna run compile && webpack && cp -r ./dist ./packages/web3",
+        "build-no-cache": "npm run bootstrap && lerna run compile --skip-nx-cache && webpack && cp -r ./dist ./packages/web3",
```

Comment:
```suggestion
        "build": "npm run bootstrap && lerna run compile --skip-nx-cache && webpack && cp -r ./dist ./packages/web3",
```

Reply:
For now, lets keep normal build command without cache as its used in lerna life cycle scripts ( like lerna version command)